Custom-Range-Slider/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Custom Range Slider</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="slider-ui color1">
+            <input type="range" min="0" max="100" step="1">
+            <div class="bar">
+                <span class="min"></span>
+                <span class="max"></span>
+            </div>
+            <div class="track">
+                <div class="value"></div>
+            </div>
+        </div>
+    
+        <div class="slider-ui color2">
+            <input type="range" min="0" max="100" step="1">
+            <div class="bar">
+                <span class="min"></span>
+                <span class="max"></span>
+            </div>
+            <div class="track">
+                <div class="value"></div>
+            </div>
+        </div>
+    
+        <div class="slider-ui color3">
+            <input type="range" min="0" max="100" step="1">
+            <div class="bar">
+                <span class="min"></span>
+                <span class="max"></span>
+            </div>
+            <div class="track">
+                <div class="value"></div>
+            </div>
+        </div>
+    </div>
+    <div  class="footer">
+        <h4>Designed & Developed with &#10084;&#65039; by Atif</h4>
+    </div>
+    <script src="./script.js"></script>
+</body>
+</html>
```

Comment:
Apply an Html beautifier to this code:
https://codebeautify.org/htmlviewer --- use this link.

Reply:
Done @chicken-biryani

Reply:
Yep its good to go now.